language-plutus-core/test/Spec.hs

```diff
@@ -56,13 +56,21 @@ compareType _ _                                      = False
 compareProgram :: Eq a => Program TyName Name a -> Program TyName Name a -> Bool
 compareProgram (Program _ v t) (Program _ v' t') = v == v' && compareTerm t t'
 
+propCBOR :: Property
+propCBOR = property $ do
+    prog <- forAll genProgram
+    let nullPosn = fmap (pure ())
```

Comment:
Didn't you just teach me this was spelled `void`?  😆

Reply:
This will be fixed :)